website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -316,6 +316,23 @@ Avoid inconsistent icon use.
 </div>
 !!!
 
+### Badge usage
+
+[Badges](/components/badge) in ListItems are optional. We recommend using only the `small` Badge option in order to more closely align with the text line height. Badges should be used sparingly and only when their use adds contextual value. Examples of appropriate use of Badge inside an Interactive ListItem include:
+
+- When an item is in beta or behind a feature flag (e.g., a new item that users should be able to easily locate).
```

Comment:
Thanks for clarifying! I will update all of these to use commas instead. I do think some of them still need to be "e.g." rather than all changing to "i.e." though